flixel/sound/FlxSound.hx

```diff
@@ -605,7 +605,7 @@ class FlxSound extends FlxBasic
 		final volume = (group != null ? group.volume : 1.0) * _volume * _volumeAdjust;
 		
 		#if FLX_SOUND_SYSTEM
-		if (FlxG.sound.muted)
+		if (FlxG.sound.muted || (group != null && group.muted))
```

Comment:
the `#if FLX_SOUND_SYSTEM` is to check is FlxG.sound exists, groups and soudns can exist without that, so this should not be checked here. group volume is check in line 605. also we should consider having group.getVolume() that checks muted and returns 0